lms/djangoapps/django_comment_client/management/commands/export_discussion_participation.py

```diff
@@ -0,0 +1,157 @@
+import csv
+import dateutil
+from datetime import datetime
+from optparse import make_option
+
+from django.core.management.base import BaseCommand, CommandError
+
+from opaque_keys import InvalidKeyError
+from opaque_keys.edx.keys import CourseKey
+from opaque_keys.edx.locations import SlashSeparatedCourseKey
+
+from courseware.courses import get_course
+from student.models import CourseEnrollment
+from lms.lib.comment_client.user import User
+import django_comment_client.utils as utils
+
+
+class _Fields:
```

Comment:
As this class isn't used anywhere else, can we just place these constants in the root namespace of this module? Flat is better than nested.